src/components/WorkspaceTabs/Review/ReviewTab.vue

```diff
@@ -95,7 +95,7 @@
             listingStyleFor(file) {
                 let common = 'px-2 py-2 flex hover:bg-grey-lighter '
                 let passive = 'bg-white'
-                let active = 'bg-blue text-white hover:bg-blue'
+                let active = 'bg-blue-600 text-white hover:bg-blue-400'
```

Comment:
@ajthinking I am curious, why the original `bg-blue` works on you? I check the tailwind docs, they don't have bg-blue class, but a bg-blue-{shadeNumber} instead.

Reply:
Youre doing it right! I installed tailwind long ago then the syntax was `bg-blue` or `bg-blue-darker` etc.